udata/harvest/commands.py

```diff
@@ -56,6 +56,15 @@ def delete(identifier):
     log.info('Deleted source "%s"', identifier)
 
 
+@grp.command()
+@click.argument('identifier')
+def clean(identifier):
```

Comment:
The name may hide what it actually does. Maybe `delete_dataset` would be more explicit?

Reply:
`delete_datasets` was my first idea, but after all I thought it's too bulky nevertheless it's good and explicit. I switched to `clean` as a presumably more elegant solution, but I'm probably not ready to die on that hill :)

For my understanding, do you think `clean` might mean something else or it's just not clear what it means?

Reply:
Agreed on the elegance consideration. ;)

I was concerned that it could be difficult to interpret what it's going to `clean`. It could also be `HarvestItem` or `HarvestJob` linked to the source for example. But items are going to be cleaned actually when datasets get deleted. So let's keep this wording with the explicit docstring :)